src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsProjectJsonToPackageReferenceMigrator.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains methods to migrate a project.json based legacy project to PackageReference based project.
+    /// </summary>
+    [ComImport]
+    [Guid("2CB0AF9B-241D-4201-99ED-00F796C416BB")]
+    public interface IVsProjectJsonToPackageReferenceMigrator
```

Comment:
You need to make sure this class and API is only available for Dev15 since it won't be applicable for Dev14